assets/js/menu.js

```diff
@@ -201,80 +203,80 @@ class Menu {
             }
         }
       });
+    });  
     const help = document.getElementById('help-container');
-    const $helpParagraph = $(help).children('p');
-    $('.side-menu, .top-widget, .lat-lng-container')
-      .on('mouseover mouseout', event => {
-        const target = event.type === 'mouseover' ? event.target : event.relatedTarget;
+    const helpParagraph = help.querySelector('p');
+    document.querySelectorAll('.side-menu, .top-widget, .lat-lng-container').forEach(element => {
+      element.addEventListener('mouseover', event => {
+        const target = event.target;
         // keep current help if pointer jumped to help container or it overgrew current pointer pos.
         if (help.contains(target)) return;
-        const helpTransId = $(target).closest('[data-help]').attr('data-help') || 'default';
-
+        const helpTransId = target.closest('[data-help]')?.getAttribute('data-help') || 'default';
         // Used only for jewelry that appears in some cycles
         if (helpTransId.includes('timestamp')) {
-          const itemId = $(target).closest('[data-help]').attr('data-type');
+          const itemId = target.closest('[data-help]')?.getAttribute('data-type');
           const timestamp = MapBase.jewelryTimestamps[itemId];
-          $helpParagraph.html(Language.get(`help.${helpTransId}`)
-            .replace('{day}', new Date(timestamp * 1000).toLocaleDateString(Settings.language, {
-              timeZone: 'UTC',
-              year: 'numeric',
-              month: 'long',
-              day: 'numeric',
-              weekday: 'long',
-            })));
+          helpParagraph.innerHTML = Language.get(`help.${helpTransId}`)
+            .replace('{day}', new Date(timestamp * 1000).toLocaleDateString(Settings.language, { timeZone: 'UTC', year: 'numeric', month: 'long', day: 'numeric', weekday: 'long' }));
           return;
         }
-
-        $helpParagraph.html(Language.get(`help.${helpTransId}`));
+        helpParagraph.innerHTML = Language.get(`help.${helpTransId}`);
+      });
+      element.addEventListener('mouseout', event => {
+        const target = event.relatedTarget;
+        if (help.contains(target)) return;
+        helpParagraph.innerHTML = '';
       });

```

Comment:
You can remove this event. It causes to not display anything in help box when mouse is over themap. If you want to remove help on mouseout, you should hide the entire help box, not only text.